apps/website/src/components/shared/form/NumberField.tsx

```diff
@@ -47,34 +75,46 @@ export function NumberField(props: NumberFieldProps) {
 
   const { locale } = useLocale();
   const state = useNumberFieldState({ ...props, locale });
-  const { labelProps, inputProps } = useNumberField(
-    customizedProps,
-    state,
-    ref,
-  );
+  const {
+    labelProps,
+    groupProps,
+    inputProps,
+    incrementButtonProps,
+    decrementButtonProps,
+  } = useNumberField(customizedProps, state, ref);
 
   return (
     <div className={props.className || "flex-1"}>
-      <label {...labelProps}>{props.label}</label>
-      <div className="flex w-full items-center gap-1 rounded-sm border border-gray-700 bg-white text-gray-500">
-        {props.prefix}
-        <input
-          className={classes(
-            "w-full flex-1 bg-white p-1 px-2 text-black",
-            props.inputClassName,
+      <label className={props.labelClassName} {...labelProps}>
+        {props.label}
+      </label>
+      <div className={classes("flex", props.groupClassName)} {...groupProps}>
+        {props.showButtons && (
+          <NumberButton {...decrementButtonProps}>-</NumberButton>
+        )}
+        <div className="flex w-full items-center justify-center rounded-sm border border-gray-700 bg-white text-gray-500 focus-within:outline">
```

Comment:
Can we make it so a click anywhere in here focuses the actual input? Visually it looks like you could click on "hours" in the form, but it doesn't do anything currently. (I guess with the exception of click on the reset x)